doc/PreventTokenReuse.md

```diff
@@ -0,0 +1,17 @@
+# Preventing TOTP Token Reuse
+
+The method `verifyOtpWithWindow` has a `after` parameter. By default, its value is `null`. A timestamp can be given to only accept token which are generated after the period of the timestamp.
+This can be used to prevent token reuse by keeping track of the last time a uses's OTP was verified. To get the exact timestamp of the code which was used the method `verifyOtpWithWindow` returns the timestamp 
```

Comment:
```suggestion
This can be used to prevent token reuse by keeping track of the last time a uses's OTP was verified successfully. To get the exact timestamp of the code which was used the method `verifyOtpWithWindow` returns the timestamp 
```